src/app/Library/CrudPanel/Traits/Tabs.php

```diff
@@ -147,18 +147,30 @@ public function getTabFields($label)
     public function getTabs()
     {
         $tabs = [];
-        $fields = $this->getCurrentFields();
+        $fields = $this->getOperationFields();
 
         $fields_with_tabs = collect($fields)
             ->filter(function ($value, $key) {
                 return isset($value['tab']);
             })
             ->each(function ($value, $key) use (&$tabs) {
-                if (! in_array($value['tab'], $tabs)) {
+                if (!in_array($value['tab'], $tabs)) {
                     $tabs[] = $value['tab'];
                 }
             });
 
         return $tabs;
     }
+
+    /**
+     * @return mixed
+     */
+    public function getOperationFields()
```

Comment:
Hmmm... I see what you did here. Sneaky sneaky!! 🤣 Ok so... I agree with what this is doing. I like the fact that Tabs has the exact same code, we don't duplicate it, we pull in Fields or Columns depending on what's actually needed by the operation. Very simple & quick solution.

I don't quite agree with the name, though. It's a bit misleading to call the method `getOperationFields` and then have it return columns - I had to read it 3 times to understand how it's working 🤣 then I noticed it. So **I propose we rename this method something else, that's more generic**... something like `getTabElements()` or `getTabItems()` or even `getCurrentFieldsOrColumns()`... I don't know... let's brainstorm.

Also, since we'll be using the `Tabs` functionality on multiple operations, and depending on the operation it returns fields/columns, **I'd rather code this in a way that Tabs are NOT dependent on the operation name in any way**, for the Tabs trait to not contain the "show" keyword. That would also allow other, custom operations, to also use the Tabs functionality if they want. We should be able to do that by moving the `$this->enableTabsIfColumnUsesThem($column);` from `Columns.php`  to the actual operation setup. Because the operation itself knows what it should be enabled on - columns/fields. Then 
- the Create&Update operations could call something like `enableTabsFor('fields')`
- the Show operation could call something like `enableTabsFor(`columns`)

That's the first solution I came up with, not sure it's the best. But if we DO make Tabs support both Fields and Columns we should definitely remove this confusion from the code and make it general, otherwise we're making it very difficult for our future selves to maintain this 😄 

What do you think about this @LemarinelNet ? Are you able to work on this a little more to make it more general, or would you rather one of us pick up the task when we get to it? Any idea for better names for these - looks like this isn't my most creative day 🤣